Dockerfile

```diff
@@ -129,9 +129,9 @@ COPY . .
 # Create script to execute firstrun.py and run it:
 ##############################################################################
 RUN echo $'#!/usr/bin/env bash \n\
-source ~/.bashrc \n\
-conda activate alltalk \n\
-python ./system/config/firstrun.py $@' > ./start_firstrun.sh
+  source ~/.bashrc \n\
+  conda activate alltalk \n\
+  python ./system/config/firstrun.py $@' > ./start_firstrun.sh
```

Comment:
This change appears to be from my format on save, if it's not desired code style I can remove. I see no issue with it for readability however.